client/client.py

```diff
@@ -131,7 +134,11 @@ def login_user(self):
         }
         url = 'https://www.skotos.net/user/login.php'
         response = requests.post(url, headers=header, data=data, allow_redirects=False)
-        # pprint(response.headers)
-        self.uname = re.search('user=(.*?);', response.headers['set-cookie']).group(1)
-        self.pwd = re.search('pass=(.*?);', response.headers['set-cookie']).group(1)
+        try:
+            pprint(response.headers)
```

Comment:
Our build doesn't typically include `pprint`. I also don't think I love leaving this in "production" code. I know this seems hypocritical. I am in the process of removing it from my code as well.